.github/ISSUE_TEMPLATE/request-for-amplification-template.yml

```diff
@@ -52,32 +77,71 @@ body:
         - "#WordCamp"
     validations:
       required: true
+  - type: dropdown
+    id: make-team
+    attributes:
+      label: MakeWP Team
+      description: What MakeWP Team is this request for?  (Select all that apply)
+      multiple: true
+      options:
```

Comment:
Do we need to add Marketing Team?
E.x. team was actively involved with Blogs to Blocks campaign. 
Someone may create request with ideas for Marketing team!

Reply:
we do!